autotest/test_igr.py

```diff
@@ -11,24 +11,23 @@
 import pytest
```

Comment:
Just an FYI that this file should be test_lgr.py (instead of test_igr.py), which stands for Local Grid Refinement.

Reply:
Got it- fixed the name